### ISSUE TITLE
Error building with Carthage

### ISSUE THREAD

**Issue by audungk** [appnexus/mobile-sdk-ios#11]
When you run `carthage update`, with a fresh project, you get this error:

```
A shell task failed with exit code 66:
2016-12-02 15:29:54.482 xcodebuild[46859:559061] warning:  The file reference for "Info.plist" is a member of multiple groups ("" and ""); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
xcodebuild: error: Scheme BuildANSDKFramework is not currently configured for the build action.
```

**Comment by audungk** [comment]
Nevermind. Carthage cache was pulling an older release. Sorry. Closing

**Comment by audungk** [comment]
Hmm... When updating, it still seems to give the same error. Will investigate, and open a PR, if I find something

**Comment by audungk** [comment]
Found the issue. This scheme: https://github.com/appnexus/mobile-sdk-ios/blob/master/BinaryProjects/ANSDK.xcodeproj/xcshareddata/xcschemes/BuildANSDKFramework.xcscheme, which is in your repo, but deleted from mine, and which doesn't seem to be used, is the one crashing carthage. Suggested fix is either to delete the scheme, or turn off sharing

**Comment by josecu** [comment]
@audungk Sorry about that, will remove the `BuildANSDKFramework.xcscheme`, this will be out in the next release in January.

**Comment by josecu** [comment]
This was fixed in 2.14, released January 2017.
